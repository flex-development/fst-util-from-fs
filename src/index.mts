/**
 * @file Package Entry Point
 * @module fst-util-from-fs
 */

export type * from '#interfaces/index'
export type * from '#types/index'
export { default as fromFileSystem } from '#util'
