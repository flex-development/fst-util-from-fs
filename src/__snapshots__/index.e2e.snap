// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:fst-util-from-fs > should expose public api 1`] = `
[
  "fromFileSystem",
]
`;
